website/layouts/language.erb

```diff
@@ -0,0 +1,1030 @@
+<% wrap_layout :inner do %>
+  <% content_for :sidebar do %>
+    <h4><a href="/docs/configuration/index.html">Terraform Language</a></h4>
+
+    <ul class="nav docs-sidenav">
+
+      <li>
+        <a href="/docs/configuration/index.html">Home</a>
```

Comment:
For language, I didn't want to name the homepage "overview" because it was going to force me to pick an unnatural name for the "Language Overview" page, which actually DOES provide an overview of the language.

In theory I'm not against it, _if_ someone can give me a non-bad name for the language overview. (Best option I tried was "language summary," but that still seemed wrong.)